test/browser/features/fixtures/plugin_vue/typescript_vue3/index.html

```diff
@@ -12,11 +12,14 @@
     <script>
       var el = document.getElementById('bugsnag-test-should-run')
       var isClassSupported = false
+      var isObjectSpreadSupported = false
       try {
         eval('class Foo extends Object {}')
         isClassSupported = true
+        eval('const a = {}; const b = { ...a }')
```

Comment:
this syntax has started to appear in the Vue bundles. This is despite it being an ECMAScript 2018 addition and Vue claiming to support all browsers that support ES2015